_apps/merge-when-ready.md

```diff
@@ -0,0 +1,15 @@
+--
+title: Merge When Ready
+description: Add a special label to merge your PR when status checks pass
+slug: mergewhenready
+screenshots:
+- https://marketplace-screenshots.githubusercontent.com/4760/cb554780-a829-11e9-8993-28186c68aaab
+- https://marketplace-screenshots.githubusercontent.com/4760/dd36ea80-a829-11e9-8fc4-3571da4aa9f0
+- https://marketplace-screenshots.githubusercontent.com/4760/ed4eca00-a829-11e9-9beb-59766847ccd5
+authors:
+- cyberhck
+repository: fossapps/Handlebars-Issue-and-Pull-Requests
```

Comment:
looks like the wrong repository?

Reply:
yes, and also, the app didn't work the way I expected, I need to update, but haven't gotten any time to do so, will try to do soon :slightly_smiling_face:

Reply:
Shall we close the PR for the time being and you reopen it once you have something ready?